.github/scripts/dockerfile-security.rego

```diff
@@ -25,7 +25,7 @@ deny[msg] {
 deny[msg] {
     input[i].Cmd == "from"
     val := split(input[i].Value[0], "/")
-    count(val) > 2
+    count(val) > 3
```

Comment:
Why are we changing this here @akihikokuroda ?

Reply:
It's a strange check.  The UBI image path is like this: `registry.access.redhat.com/ubi9/python-311` This path should be trusted but the lint fails.

Reply:
Yeah, I'm not sure how exactly the number of slashes corresponds to trustworthiness but que sera sera...

Reply:
Haha understood 👍